lerna.json

```diff
@@ -10,7 +10,7 @@
     "publish": {
       "forcePublish": true,
       "skipGit": true,
-      "registry": "https://registry.npmjs.org/"
+      "registry": "http://localhost:4873/"
```

Comment:
Is this change intentional?

Reply:
Good catch. No that chage was on accident and is a remainder from local testing 😅